File: pubnub-gson/pubnub-gson-api/src/main/java/com/pubnub/api/java/endpoints/channel_groups/RemoveChannelChannelGroup.java
package com.pubnub.api.java.endpoints.channel_groups;

import com.pubnub.api.java.endpoints.Endpoint;
import com.pubnub.api.models.consumer.channel_group.PNChannelGroupsRemoveChannelResult;

public interface RemoveChannelChannelGroup extends Endpoint<PNChannelGroupsRemoveChannelResult> {
    RemoveChannelChannelGroup channelGroup(String channelGroup);

    RemoveChannelChannelGroup channels(java.util.List<String> channels);
}

File: pubnub-kotlin/pubnub-kotlin-api/src/jvmMain/kotlin/com/pubnub/api/endpoints/channel_groups/RemoveChannelChannelGroup.kt
package com.pubnub.api.endpoints.channel_groups

import com.pubnub.api.Endpoint
import com.pubnub.api.models.consumer.channel_group.PNChannelGroupsRemoveChannelResult

/**
 * @see [PubNub.removeChannelsFromChannelGroup]
 */
interface RemoveChannelChannelGroup : com.pubnub.kmp.endpoints.channel_groups.RemoveChannelChannelGroup, Endpoint<PNChannelGroupsRemoveChannelResult> {
    val channelGroup: String
    val channels: List<String>
}
